cmd/libs/go2idl/generator/import_tracker.go

```diff
@@ -44,48 +49,110 @@ func NewImportTracker(types ...*types.Type) *ImportTracker {
 			// Add other forbidden keywords that also happen to be
 			// package names here.
 		},
+		lang: "golang",
 	}
 	tracker.AddTypes(types...)
 	return tracker
 }
 
+func NewProtobufImportTracker(local types.Name, types ...*types.Type) *ImportTracker {
```

Comment:
I think I kind of agree with @smarterclayton, that just having two implementations of a common interface would be ideal. proto is pretty similar (I guess) but other languages might be very different?